src/DependencyInjection/Admin/TaggedAdminInterface.php

```diff
@@ -41,25 +42,27 @@
  *     - The first and third argument are automatically injected by the AddDependencyCallsCompilerPass.
  *     - The second one is used as a reference of the Admin in the Pool, with the `setAdminClasses` call.
  *
- * @method void                          initialize()
- * @method void                          setLabel(?string $label)
- * @method void                          showMosaicButton(bool $isShown)
- * @method void                          setPagerType(string $pagerType)
- * @method string                        getPagerType()
- * @method void                          setManagerType(string $managerType)
- * @method void                          setSecurityInformation(array $information)
- * @method void                          setFilterPersister(?FilterPersisterInterface $filterPersister = null)
- * @method FilterPersisterInterface|null getFilterPersister()
- * @method bool                          hasFilterPersister()
- * @method void                          setModelManager(ModelManagerInterface $modelManager)
- * @method void                          setDataSource(DataSourceInterface $dataSource)
- * @method DataSourceInterface           getDataSource()
- * @method FormContractorInterface       getFormContractor()
- * @method void                          setShowBuilder(ShowBuilderInterface $showBuilder)
- * @method ShowBuilderInterface          getShowBuilder()
- * @method Pool                          getConfigurationPool()
- * @method void                          setRouteGenerator(RouteGeneratorInterface $routeGenerator)
- * @method RouteGeneratorInterface       getRouteGenerator()
+ * @method void                             initialize()
```

Comment:
All of these method annotations look weird 🧐

Reply:
why ?

Reply:
Just an idea for all these methods...

Can we add a new `NextMajorTaggedAdminInterface` with all these methods and mark it as `@internal`. 

We can then use `@mixin NextMajorTaggedAdminInterface` to include all these methods.